doc/constantdict.rst

```diff
@@ -3,6 +3,8 @@ constantdict
 
 .. autoclass:: constantdict.constantdict
 
+.. autoclass:: constantdict.constantmutabledict
```

Comment:
Document what can be done with this.

Reply:
Done in 2e054bd2b3614c3863e26cab833c056aeeba8e05, 8081a35ede0b342b5b7f915b37199d57f2db6a91